The sleeps are necessary only with the Novell package. It needs some
more investigation why our dialog is slower.

BTW: Lado Michnovic was not able to reproduce it. Though, he probably did not
export OOO_FORCE_SYSALLOC=1
export MALLOC_CHECK_=2

It is strange that it does not happen with the Sun package

--- graphics/options/inc/global/id_opt_1.inc.orig	2006-05-15 19:29:55.000000000 +0200
+++ graphics/options/inc/global/id_opt_1.inc	2006-06-01 17:23:50.000000000 +0200
@@ -679,6 +679,7 @@
 
    Kontext "ExtrasOptionenDlg"
    ExtrasOptionenDlg.OK
+   sleep (2)
 
   printlog ("'/// - check if all UnChecked ///")
    ToolsOptions
@@ -743,6 +744,7 @@
 
    Kontext "ExtrasOptionenDlg"
    ExtrasOptionenDlg.OK
+   sleep (2)
 
   printlog ("'/// - check if all Checked ///")
    ToolsOptions
